InstallGuide.md

```diff
@@ -1,5 +1,7 @@
-# Up to date installation directions and manifests can be found in the [Calico docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal)
+# [Canal docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal) can now be found in the [Calico docs](https://docs.projectcalico.org/)
```

Comment:
"can now be found on docs.projectcalico.org" reads a bit better to me.

Reply:
Please see if you like what I've got now. A bit of a combination of your tweak with something suggested by @ozdanborne